tensorly/contrib/sparse/backend/numpy_backend.py

```diff
@@ -0,0 +1,144 @@
+from copy import copy as _py_copy
+from distutils.version import LooseVersion
+
+import numpy as np
+import scipy
+import scipy.linalg
+import scipy.sparse.linalg
+import sparse
+
+from . import register_sparse_backend
+from ....backend.core import Backend
+
+
+_MIN_SPARSE_VERSION = '0.4.1+10.g81eccee'
+if LooseVersion(sparse.__version__) < _MIN_SPARSE_VERSION:
+    raise ImportError("numpy sparse backend requires `sparse` version >= %r"
+                      % _MIN_SPARSE_VERSION)
+
+
+def is_sparse(x):
+    return isinstance(x, sparse.COO)
```

Comment:
This should be `sparse.SparseArray` to ensure that all formats are supported. I may improve `DOK` support and also add more formats.